README.md

```diff
@@ -57,8 +57,10 @@ Create a virtualenv into which you can install the module.
 
 Run the script with the Fedora configuration::
 
-    $ ./asknot-ng.py templates/index.html questions/fedora.yml l10n/fedora/locale --theme fedora
-    Wrote build/en/index.html
+    $ ./asknot-ng.py templates/index.html questions/arch.yml l10n/arch/locale --theme arch
+    
+    
+Wrote build/en/index.html
```

Comment:
Should keep this line indented with the command as this is part of the command's output.